gpu/command_buffer/service/common_decoder.cc

```diff
@@ -359,7 +369,29 @@ error::Error CommonDecoder::HandleGetBucketData(uint32_t immediate_data_size,
       return error::kInvalidArguments;
   }
   memcpy(data, src, size);
+#if defined(CASTANETS)
+  scoped_refptr<gpu::Buffer> buffer =
+      command_buffer_service_->GetTransferBuffer(args.shared_memory_id);
+  mojo::SyncSharedMemoryHandle(buffer->backing()->GetGUID(), offset, size);
+#endif
+  return error::kNoError;
+}
+
+#if defined(CASTANETS)
+error::Error CommonDecoder::HandleGetResultData(uint32_t immediate_data_size,
```

Comment:
How about naming this method SyncResultData()? In Castanets, we already use Sync*() kind of names for sync methods.

Reply:
I agree. I updated.